etc/config.sample.toml

```diff
@@ -67,6 +67,10 @@
   # log any sensitive data contained within a query.
   # query-log-enabled = true
 
+  # Validates incoming writes to ensure keys only have valid unicode characters.
+  # This setting will incur a small overhead because every key must be checked.
```

Comment:
nit: I would remove `small`; I think the overhead could be significant depending on the key length for example.